cloud/src/definer/crud.definer.js

```diff
@@ -148,18 +148,42 @@ Parse.Cloud.define('postObjectsToClass', (request) => new Promise((resolve, reje
     localObject - Continas key value pairs that will be posted to the class
                 - this contains a latitude/longitude which will post the location
     parseParentClassID - ID of the parseParentClass Object to associate the new post with
+    loopParentID - Custom Form that the looped form originates from
+    loop - bool t/f whether looped data contained in form
   ******************************************* */
 Parse.Cloud.define('postObjectsToClassWithRelation', (request) => new Promise((resolve, reject) => {
   const supplementaryForm = new Parse.Object(request.params.parseClass);
   const residentIdForm = new Parse.Object(request.params.parseParentClass);
   const userObject = new Parse.Object('_User');
+  const loopParentForm = new Parse.Object(request.params.parseClass);
 
   // Create supplementaryForm points
-  const { localObject } = request.params;
-  for (const key in localObject) {
+  const { localObject, loop } = request.params;
+  // create looped json to ensure that data is submitted as multiple forms
+  const loopedJson = {};
+  let newFieldsArray = [];
+  Object.keys(localObject).forEach((key) => {
     const obj = localObject[key];
     if (!obj.includes('data:image/jpg;base64,')) {
-      supplementaryForm.set(String(key), obj);
+      if (loop === true && String(key) === 'fields') {
+        obj.forEach((field, index) => { // eslint-disable-line
+          if (!field.title.includes('__loop')) {
+            newFieldsArray = newFieldsArray.concat(field);
+          } else {
+            const originalKey = field.title.split('__loop');
+            // check if loop already used
+            if (originalKey[1] in Object.keys(loopedJson)) {
+              loopedJson[originalKey[1]][originalKey[0]] = field.answer;
+            } else {
+              loopedJson[originalKey[1]] = {};
+              loopedJson[originalKey[1]][originalKey[0]] = field.answer;
+            }
+          }
+        });
```

Comment:
I want to move his loop logic out of this `crud.definer`, could you move this logic to a folder named `_utils` and move this logic there?